### ISSUE TITLE
App crash on close in gnome

### ISSUE THREAD

**Issue by gxanshu** [alainm23/planify#1495]
Thank you for creating such a beautiful and helpful Todo app. i really appreciate your work

**Describe the bug**
when we close the app gnome displays an error as app has been crashed

**To Reproduce**
1. open the app
2. do some activities
3. close the app
4. you will see an error notification from gnome

**Expected behavior**
it should just close perfectly

**Screenshots**
![image](https://github.com/user-attachments/assets/94b5cafe-21bd-47e2-8007-9b6ddc429fe6)
![image](https://github.com/user-attachments/assets/c12c20ab-5cd4-4b1d-ae70-37f1dc938340)

**Desktop (please complete the following information):**
 - OS or DE: Fedora 41
 - Version: 4.11.6

**Additional context**
Not sure if the problem is in my gnome or in within the app.

**Comment by JStrategic** [comment]
Been having this issue as well on Fedora 41 but on KDE. I think it may be a fedora thing not this app specifically, as I also have a couple of other flatpak apps that do the same thing.

**Comment by gxanshu** [comment]
yeah it can be, after that issue i also saw same errors from other flatpack apps

**Comment by alainm23** [comment]
Hi, it’s been a while since this issue was created — have you seen the error happen again?
I’ll go ahead and close this for now, but feel free to reopen it if the problem persists.
